simulator-ui/src/main/resources/static/src/app/components/scenarios/launch/scenario-launch.ts

```diff
@@ -30,18 +30,18 @@ export class ScenarioLaunchComponent implements OnInit {
 
     getScenario(name: string) {
         this.scenarioService.getScenario(name)
-            .subscribe(
-                scenario => this.scenario = scenario[0],
-                error => this.errorMessage = <any>error
-            );
+            .subscribe({
+                next: (scenario) => this.scenario = scenario[0],
+                error: (error) => this.errorMessage = <any>error
+            });
     }
 
     getScenarioParameters(name: string) {
         this.scenarioService.getScenarioParameters(name)
-            .subscribe(
-                scenarioParameters => this.scenarioParameters = scenarioParameters,
-                error => this.errorMessage = <any>error
-            );
+            .subscribe({
+                next: (scenarioParameters) => this.scenarioParameters = scenarioParameters,
+                error: (error) => this.errorMessage = <any>error
```

Comment:
interesting cast on all those `erros` - is that really required? a `toString` would maybe be more applicable. can you please double check @luethiandrin?